packages/terra-clinical-item-collection/src/Item.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ItemView from 'terra-clinical-item-view';
+import List from 'terra-list';
+import Table from 'terra-table';
+import 'terra-base/lib/baseStyles';
+import styles from './ItemCollection.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The display elements to be presented.
+   */
+  children: PropTypes.node,
+  /**
+   * The start accessory element to be presented.
+   */
+  startAccessory: PropTypes.node,
+  /**
+   * The comment element to be presented.
+   */
+  comment: PropTypes.node,
+  /**
+   * The end accessory element to be presented.
+   */
+  endAccessory: PropTypes.node,
+  /**
+   * The styles to apply to the ItemView when represented as a list item. Styles options are layout, textEmphasis,
+   * isTruncated and accessoryAlignment.
+   */
+  listItemStyles: PropTypes.shape({
+    layout: PropTypes.oneOf(['oneColumn', 'twoColumns']),
+    textEmphasis: PropTypes.oneOf(['default', 'start']),
+    isTruncated: PropTypes.bool,
+    accessoryAlignment: PropTypes.oneOf(['alignTop', 'alignCenter']),
+  }),
+  /**
+   * Wether or not the item is selectable. If true, the item is given list and table hover and focus styles, tabIndex
+   * set to 0, and onClick and onKeyDown callbacks set to the onSelect function provided via Item Collection.
+   */
+  isSelectable: PropTypes.bool,
+  /*
+   * Wether or not the item is selected.
+   */
+  isSelected: PropTypes.bool,
+  /**
+   * The comment element to be presented.
+   */
+  view: PropTypes.oneOf(['list', 'table']),
+};
+
+const defaultProps = {
+  startAccessory: undefined,
+  comment: undefined,
+  endAccessory: undefined,
+  listItemStyles: {},
+  isSelectable: false,
+  isSelected: false,
+  view: 'list',
+};
+
+function createSelectableProps(onSelect, itemKey) {
+  const selectableProps = { isSelectable: true };
+  selectableProps.tabIndex = 0;
+
+  if (!itemKey) {
+    /* eslint-disable no-console */
+    console.error('Key is required for correct selectable implementation.');
+    /* eslint-disable no-console */
+  }
+
+  if (onSelect) {
+    selectableProps.onClick = (event) => {
+      onSelect(event, itemKey);
+    };
+
+    selectableProps.onKeyDown = (event) => {
+      onSelect(event, itemKey);
+    };
+  }
+
+  return selectableProps;
+}
+
+function createListItem(elements, itemKey, selectableProps, isSelected, listItemStyles) {
+  const { startAccessory, children, comment, endAccessory, hasStartAccessory, hasEndAccessory, ...listItemProps } = elements;
+  // hasStartAccessorySpacing={hasStartAccessory}
```

Comment:
This will be hooked into #217 once completed. May or may not be the prop names we'd expect but wanted it set up.